cmd/infracost/diff.go

```diff
@@ -64,8 +66,8 @@ func checkDiffConfig(cfg *config.Config) error {
 			return errors.New("terraform_use_state cannot be used with `infracost diff` as the Terraform state only contains the current state")
 		}
 
-		if projectConfig.TerraformParseHCL && cfg.CompareTo == "" {
-			return errors.New("Use `infracost diff --path /code --terraform-parse-hcl --compare-to infracost-previous-run.json` to generate a diff")
+		if projectConfig.ProjectType == "terragrunt_dir" && cfg.CompareTo == "" {
```

Comment:
should we also include `terraform_dir` case here? Or is that already covered?

Reply:
Cheers, looks like this actually had a bug, so have fixed it and added a test.

Reply:
👍